api/src/routes/wish.js

```diff
@@ -127,7 +129,7 @@ wishRouter
       req.body,
       { new: true },
       (err, wish) => {
-        if (err) return res.status(500).send(err)
+        if (err) return res.status(409).send(err)
```

Comment:
Hi @kidCaulfield, thank you for the contribution. Can you add a test to capture this error case. I realize we might not have had one for a 500 error, but we appreciate helping fill these gaps.

Reply:
I can put the 500 check back in.